### DISCUSSION TITLE
How to get debug output (manifest mode)?

### DISCUSSION THREAD

**Discussion by thxkiwi** [microsoft/vcpkg#29982] [Q&A]
I see a lot of logs in the Issues section have output that provides "[DEBUG]" messages.

How do I get this and any other debugging output from vcpkg? 

Based on other discussions and stackoverflow posts, I've tried the following environment variables in Windows
* VCPKG_DEBUG=1
* VCPKG_INSTALL_OPTIONS=--debug

and don't get anything other than the usual output.

**Comment by autoantwort** [comment]
Pass `--debug` as an argument to vcpkg

**Reply by thxkiwi** [reply]
How does one pass arguments when vcpkg is executed by msbuild?

**Reply by autoantwort** [reply]
https://learn.microsoft.com/en-us/vcpkg/users/buildsystems/msbuild-integration#vcpkg-additional-install-options
